commodore/cluster.py

```diff
@@ -50,6 +50,8 @@ def render_target(target: str, components: Iterable[str]):
         if defaults_file.is_file():
             classes.append(f"defaults.{component}")
 
+    # TODO Remove before merge!
+    classes.append('common')
```

Comment:
This needs to be included after `global.common`

Reply:
Resolved by splitting class includes and parameters into different files.